github/pulls.go

```diff
@@ -134,6 +135,29 @@ func (s *PullRequestsService) Get(owner string, repo string, number int) (*PullR
 	return pull, resp, err
 }
 
+// Get pull request raw format (diff or patch)
+func (s *PullRequestsService) GetRaw(owner string, repo string, number int, opt *MediaRawTypeOption) (string, *Response, error) {
```

Comment:
I think `MediaRawTypeOption` should be named `GetRawOptions`. See other funcs that take an options struct, that's the pattern they follow.